Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Stage 1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 as build
+WORKDIR /build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY ./Dfe.Academies.Academisation.Core/ ./Dfe.Academies.Academisation.Core/
+COPY ./Dfe.Academies.Academisation.Core.Test/ ./Dfe.Academies.Academisation.Core.Test/
```

Comment:
Since we're not publishing the test projects. It might be worth removing any of the projects suffixed with Test.

Reply:
I did try removing those, but I think because they're referenced in the solution file, `dotnet restore` fails without them
They don't end up in the final image though - only the published `/app` directory

Reply:
ah, you could try - `RUN dotnet restore Dfe.Academies.Academisation.WebApi`
Would probably need to do the same on the build. This is assuming you were intending to use a different docker file for running the tests.

Reply:
Ah k, yup I'll give that a go 👍

Reply:
Yeah, I usually restore the solution then build the api proj when creating the release build

Reply:
This is done now 👍 
It also fixed some other issues, so I assume this is how we were supposed to be doing it
(eg. it was listening on the wrong port, and wasn't being exposed correctly via docker)